src/app/resolvers/tech-record-validate/tech-record-validate.resolver.ts

```diff
@@ -76,6 +80,7 @@ const handlePsv = (record: TechRecordVehicleType<'psv'>) => {
 
 const handleTrl = (record: TechRecordVehicleType<'trl'>) => {
   const validatedRecord: TechRecordVehicleType<'trl'> = { ...record };
+  validatedRecord.techRecord_vehicleClass_description = VehicleClassDescription.Trailer;
```

Comment:
Can't decide if I want to check if it is a trailer class before adding. AC is trailer must always be a trailer class

Reply:
Probably fine as is, self documenting for trailers, the only considerations are if it would be inherently clear that this method would default the description and if anything would ever need it to not do that.

Reply:
Good point, once all the validations are added this sprint, can come back to make this resolver more clear if needed